FirebaseDatabase/Sources/Core/FSyncPoint.m

```diff
@@ -161,14 +161,9 @@ - (FView *)getView:(FQuerySpec *)query
     return [[FView alloc] initWithQuery:query initialViewCache:viewCache];
 }
 
-/**
- * Add an event callback for the specified query
- * Returns an array of events to raise.
- */
-- (NSArray *)addEventRegistration:(id<FEventRegistration>)eventRegistration
-       forNonExistingViewForQuery:(FQuerySpec *)query
-                      writesCache:(FWriteTreeRef *)writesCache
-                      serverCache:(FCacheNode *)serverCache {
+- (void)registerQuery:(FQuerySpec *)query
```

Comment:
This method does everything that `addEventRegistration` did before actually adding an event registration to the view.